### PR TITLE
Danwt/remove duplicate imports

### PR DESCRIPTION
**Pull request by danwt**
The linter was complaining.

### CODE REVIEW

File: x/ccv/provider/keeper/genesis_test.go [hunk 1]
```diff
@@ -11,7 +11,6 @@ import (
 	consumertypes "github.com/cosmos/interchain-security/x/ccv/consumer/types"
 	"github.com/cosmos/interchain-security/x/ccv/provider/keeper"
 	"github.com/cosmos/interchain-security/x/ccv/provider/types"
-	providertypes "github.com/cosmos/interchain-security/x/ccv/provider/types"
```
Comment (mpoke) [inline x/ccv/provider/keeper/genesis_test.go:14]: Personally, I would have prefer to keep `providertypes` than `types`, as it's more descriptive.

**Review by mpoke — APPROVED** [review]
LGTM. See my comment below.

File: x/ccv/provider/keeper/genesis_test.go (hunk 1 above)
Comment (MSalopek) [reply x/ccv/provider/keeper/genesis_test.go:14]: I agree with Marius on this one

**Review by MSalopek — APPROVED** [review]
Looks good.
A small nitpick would be to maybe use
`providertypes "github.com/cosmos/interchain-security/x/ccv/provider/types"` instead of
`"github.com/cosmos/interchain-security/x/ccv/provider/types"` since it is more apparent what pkg is used in the code.

**Comment by danwt** [comment]
Out of date

**Pull request closed without merging**